addons/l10n_ch/tests/test_swissqr.py

```diff
@@ -183,3 +184,32 @@ def test_swissQR_qriban(self):
         self.invoice1.partner_bank_id = qriban_account
         self.invoice1.action_post()
         self.swissqr_generated(self.invoice1, ref_type="QRR")
+
+    def test_swiss_order_reference_isr_for_qr_code(self):
+        if 'sale.order' not in self.env:
```

Comment:
instead of creating a whole module for one test and in order to reduce the diff, I summon the `skipTest` if Sale is not installed